Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -83,6 +83,10 @@
   // https://github.com/dart-lang/webdev/blob/master/webdev/pubspec.yaml#L11
   public static final String MIN_WEBDEV_SDK_VERSION = "2.6.0";
 
+  // The dart cli command (https://dart.dev/tutorials/server/get-started) provides a language server command, `dart language-server` which
+  // should be used going forward instead of `dart .../analysis_server.dart.snapshot`.
+  public static final String MIN_DARTDEV_LANG_SERVER_SDK_VERSION = "2.16.0";
```

Comment:
I think this can be `2.14.4` (see https://github.com/Dart-Code/Dart-Code/blob/3fa9e24e0486785cab67e3d69e32e5026f50fca5/src/shared/capabilities/dart.ts#L23).

Reply:
Jaime says:
> the Dart SDK of 2.16 is a bit arbitrary, and I intentionally left support for older SDK as we have done in the past.

I personally have no opinion. I guess it doesn't matter.

Reply:
It shouldn't, I tested back to `2.16.0`.